code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -404,6 +412,14 @@
 				return 1
 			else
 				return 0
+	else if(istype(M, /mob/dead/observer))
+		var/mob/dead/observer/O = M
+		if(O.data_hud_seen == DATA_HUD_SECURITY_ADVANCED || O.data_hud_seen == DATA_HUD_DIAGNOSTIC + DATA_HUD_SECURITY_ADVANCED + DATA_HUD_MEDICAL_ADVANCED)
+			switch(hudtype)
+				if("security")
+					return 1
```

Comment:
TRUE and FALSE below. Best to use those defines if you're using a boolean value

Reply:
Check, thanks!